src/java.base/share/classes/sun/security/ec/ECOperations.java

```diff
@@ -685,12 +692,13 @@ protected void verifyTables(PointMultiplier multiplier) {
                         ArrayUtil.reverse(b);
                         System.arraycopy(b, 0, s, 0, b.length);
 
-                        // Compare this multiplier to the table (generated by Default multiplier)
+                        // Compare this multiplier to the table
+                        // (generated by Default multiplier)
                         AffinePoint m = multiplier.pointMultiply(s).asAffine();
                         AffinePoint v = points[d][w].asAffine();
                         if (!m.equals(v)) {
                             java.util.HexFormat hex = java.util.HexFormat.of();
-                            throw new RuntimeException("Bad multiple found at ["+d+"]["+w+"]" + hex.formatHex(s) + " " + m.getX().asBigInteger());
+                            throw new RuntimeException();
```

Comment:
I think your cleanup went to far. You should have some message saying they are not equal and if you don't want to print hex, remove getting an instance.

Reply:
I put the message back.. I removed it 'half'-intentionally; Was comparing against the original version and it didn't have  any details, thought maybe should follow suit. But I did find this message helpful, so its back.